### PR TITLE
Fixed Express method overloading

### PR DESCRIPTION
**Pull request by SyncroIT**
- Fixed express method overloading (redirect method)
- Corrected “middleware” in “middleware”

### CODE REVIEW

File: definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: mixed & {[name: string]: string};
```
Comment (bassettsj) [inline definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js]: I think it should actually just be ` {[name: string]: mixed};` I was going to change this myself, thanks for fixing it.

**Comment by bassettsj** [comment]
🙏 Thank you! Seems like good changes, @SyncroIT.  Could you fix the  merge conflicts and the `req.locals`?

File: definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js (hunk 1 above)
Comment (SyncroIT) [inline definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js]: Mmm.. yep, I'm thinking that it is as you say, I will change it tomorrow, or in the worst case, on Thursday.

**Comment by SyncroIT** [comment]
@bassettsj HI Steven, I don't see the merge conflicts, can you make me a screenshot of conflicts or tell me how to see them? 
Do you mean line 15 of express_v4.x.x.js with `req.locals`? If not, what do you mean with it?
Thanks in advance, I will fix it tomorrow or Thursday

**Comment by bassettsj** [comment]
This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests.

**Comment by SyncroIT** [comment]
:+1:  I will try to resolve the conflicts, it's the first time that I need to resolve a conflict on GitHub, sorry. Before this I worked on Mercurial, but I will try.

**Comment by marudor** [comment]
Fixed with https://github.com/flowtype/flow-typed/pull/202

**Pull request closed without merging**